src/DonationForms/resources/registrars/templates/fields/Phone.tsx

```diff
@@ -2,17 +2,17 @@ import InputMask from 'react-input-mask';
 
 import {PhoneProps} from '@givewp/forms/propTypes';
 
-export default function Phone({Label, ErrorMessage, fieldError, description, phoneFormat, inputProps}: PhoneProps) {
+export default function Phone({Label, ErrorMessage, placeholder, description, phoneFormat, inputProps}: PhoneProps) {
```

Comment:
This is causing a typescript compile error (`npm run dev`)
<img width="1323" alt="Screenshot 2023-08-31 at 3 13 18 PM" src="https://github.com/impress-org/givewp/assets/10138447/7fdc38df-e17d-4c3c-a866-a98bcb163e5e">

Reply:
Resolved in b870e65a4